Runtime/Core/ApiGatewayManagement/AmazonGameLiftWrapper.cs

```diff
@@ -71,6 +71,26 @@ public Task<CreateFleetResponse> CreateFleet(CreateFleetRequest request)
         {
             return _amazonGameLiftClient.CreateFleetAsync(request);
         }
+
```

Comment:
Consider adding a separate region, `infrastructure`, for these methods since they are not meant for the server or client builds

Reply:
As the comment for the region says, these calls should all be done in editor, so maybe its better to renaming the existing region?

Reply:
Hmm okay, maybe update comment to something like `Server region is code dedicated to Amazon GameLift SDK and AWS SDK calls made by the game server or by the Editor to prepare the server infrastructure.`

Reply:
Can be done in a follow-up, and I can't use the suggestion tool to solve this myself